commands/scan/dockerscan.go

```diff
@@ -63,17 +68,19 @@ func (dsc *DockerScanCommand) Run() (err error) {
 	}()
 
 	// Run the 'docker save' command, to create tar file from the docker image, and pass it to the indexer-app
-	if dsc.progress != nil {
-		dsc.progress.SetHeadlineMsg("Creating image archive 📦")
-	}
-	log.Info("Creating image archive...")
-	imageTarPath := filepath.Join(tempDirPath, "image.tar")
-	dockerSaveCmd := exec.Command("docker", "save", dsc.imageTag, "-o", imageTarPath)
-	var stderr bytes.Buffer
-	dockerSaveCmd.Stderr = &stderr
-	err = dockerSaveCmd.Run()
-	if err != nil {
-		return fmt.Errorf("failed running command: '%s' with error: %s - %s", strings.Join(dockerSaveCmd.Args, " "), err.Error(), stderr.String())
+	imageTarPath := dsc.imageTag
+	if !dsc.isTar {
```

Comment:
shouldn't we want to keep the regular flow as un-tabbed as possible?
if it is Tar - and we do... nothing? shouldn't we want to log it at least?

Reply:
The UI prints indexing filename.tar so it is logged to the screen:
![Screenshot 2024-02-25 at 21 55 21](https://github.com/jfrog/jfrog-cli-security/assets/144031599/634fd36b-9ad4-423b-b977-80b591b0ede5)